content/enterprise_features/s2i.md

```diff
@@ -348,3 +348,10 @@ spec:
      --all \
      --reference-policy=local
  ```
+
+### sakuli.properties does not work
+
+Instead of defining a Sakuli test suite in TESTSUITE_CONTEXT_DIR, you can specify your Sakuli project.
+This will create an S2I container containing the desired 'sakuli.properties' file. Since a Sakuli project is used
```

Comment:
```suggestion
This will create an S2I container containing the 'sakuli.properties' file included in the cloned project. Since a Sakuli project is used
```